specification.rst

```diff
@@ -1300,7 +1300,8 @@ values set on the ``cs:name`` and ``cs:et-al`` child elements of the original
 ``cs:names`` element, may also be used. If ``cs:substitute`` contains multiple
 child elements, the first element to return a non-empty result is used for
 substitution. Substituted variables are suppressed in the rest of the output to
-prevent duplication. An example, where an empty "author" name variable is
+prevent duplication. If the variable was already rendered before substitution,
```

Comment:
```suggestion
prevent duplication. If the variable was rendered outside the context of the substitution,
```

Reply:
This change seems like it would apply to a variable rendered after the substitute element.

How about `If the variable was rendered earlier in the citation, before the "substitute" element,`?

Reply:
I think that helps; thanks.

Merged.